openfisca_us/variables/irs/intermediate.py

```diff
@@ -211,6 +211,29 @@ def formula(tax_unit, period, parameters):
         return max_(tax_unit("c00100", period), 0)
 
 
+class hasqdivltcg(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Has qualified dividends or long-term capital gains"
+    documentation = "Whether this tax unit has qualified dividend income, or long-term capital gains income"
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        INCOME_SOURCES = [
+            "c01000",
+            "c23650",
+            "filer_p23250",
+            "filer_e01100",
+            "filer_e00650",
+        ]
+        return np.any(
```

Comment:
Unless there are negatives this could simplify. If it does need to capture negatives could you add a comment?
```suggestion
        return add(tax_unit, period, INCOME_SOURCES) > 0
```